packages/nx/src/command-line/generate.ts

```diff
@@ -235,7 +235,7 @@ async function convertToGenerateOptions(
   }
 
   if (!collectionName) {
-    throwInvalidInvocation(['@nrwl/workspace:library']);
+    throwInvalidInvocation(['@nrwl/js:library']);
```

Comment:
Is this working as expected?

Reply:
Actually not, the only way we might not have a `collectionName` is by running a `workspace-generator`, which at the moment fails anyways with:
```
 >  NX   Could not find any generators named "my-generator"
```

I will remove that block

cc @AgentEnder should this still be possible or should the `workspace-generator` generator be removed in v16?:
1. npx nx generate @nrwl/workspace:workspace-generator my-generator
2. npx nx generate my-generator --dry-run
3. Fails with the above error